src/apps/content-editor/src/app/views/ItemList/ItemList.js

```diff
@@ -190,24 +188,26 @@ export default connect((state, props) => {
       ]);
       if (_isMounted.current) {
         // render 1st page of results
-        // HACK: runFilters after waiting for new props
+        // HACK: runFilters next render cycle with fresh props/state
```

Comment:
Still not sure I'm following why this is a "hack" as it is the default behavior on setting state. **Unless I'm missing something**.

If the intent is to document the flow of this code as it can be complex for those learning React I would suggest these all get moved up to where the state is initialized and describe the flow without noting it is a hack. Also include why we are describing the flow that is standard behavior. This should also help with the repetition of these comments.